playbooks/device_credential_workflow_manager.yml

```diff
@@ -0,0 +1,107 @@
+- hosts: dnac_servers
+  vars_files:
+    - credentials.yml
+  gather_facts: no
+  connection: local
+  tasks:
+#
```

Comment:
You can remove these comments. I don't see any use of it.. Also we can remove all unwanted comments and blank lines...

```
- hosts: dnac_servers
  vars_files:
    - credentials.yml
  gather_facts: no
  connection: local
  tasks:

    - name: Create Credentials and assign them to a site
      cisco.dnac.device_credential_workflow_manager:
        dnac_host: "{{ dnac_host }}"
        dnac_port: "{{ dnac_port }}"
        dnac_username: "{{ dnac_username }}"
        dnac_password: "{{ dnac_password }}"
        dnac_verify: "{{ dnac_verify }}"
        dnac_debug: "{{ dnac_debug }}"
        dnac_log: True
        state: merged
        config:
          - global_credential_details:
              cli_credential:
                - description: CLI1
                  username: cli1
                  password: '12345'
                  enable_password: '12345'
              snmp_v2c_read:
                - description: SNMPv2c Read1
                  read_community: '123456'
              snmp_v2c_write:
                - description: SNMPv2c Write1
                  write_community: '123456'
              snmp_v3:
                - auth_password: '12345678'
                  auth_type: SHA
                  snmp_mode: AUTHPRIV
                  privacy_password: '12345678'
                  privacy_type: AES128
                  username: snmpV31
                  description: snmpV31
              https_read:
                - description: HTTP Read1
                  username: HTTP_Read1
                  password: '12345'
                  port: 443
              https_write:
                - description: HTTP Write1
                  username: HTTP_Write1
                  password: '12345'
                  port: 443
          assign_credentials_to_site:
            cli_credential:
              description: CLI
              username: cli
            snmp_v2c_read:
              description: SNMPv2c Read
            snmp_v2c_write:
              description: SNMPv2c Write
            snmp_v3:
              description: SNMPv3
            https_read:
              description: HTTP Read
              username: HTTP_Read
            https_write:
              description: HTTP Write
              username: HTTP_Write
            site_name:
              - Global/Chennai/Trill
              - Global/Chennai/Tidel
```